### DISCUSSION TITLE
[BUG] - enable ingredient amounts by default

### DISCUSSION THREAD

**Discussion by realthore** [mealie-recipes/mealie#4033] [General]
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

Basically as the title says. Would be neat to have some global setting to toggle that enabled the ingredient amounts for all recipes. 

### Steps to Reproduce

n/a

### Please provide relevant logs

n/a

### Deployment

Docker (Synology)

**Comment by Kuchenpirat** [comment]
check your group settings. 
They can be accessed by clicking on your profile pic in the top left and then group settings. 

I don't remember how it is called exactly but you should be able to figure it out.

**Comment by michael-genson** [comment]
This is already a feature. The default behavior is in group settings:
![image](https://github.com/user-attachments/assets/140edba2-cae4-438a-bfcc-bd3c94b61802)

And you can do it in bulk for existing recipes from recipe management:
![image](https://github.com/user-attachments/assets/7527b146-b9f1-4423-9cec-6682daa3be2b)
